tests/test.ts

```diff
@@ -0,0 +1,23 @@
+/// <reference path="typings/mocha/mocha.d.ts" />
+/// <reference path="typings/chai/chai.d.ts" />
```

Comment:
Instead of using `/// <reference path=... />`s, include these into the `tsconfig.json`.